docs/features/CONTROLNET.md

```diff
@@ -96,6 +94,8 @@ A model that generates normal maps from input images, allowing for more realisti
 **Image Segmentation**: 
 A model that divides input images into segments or regions, each of which corresponds to a different object or part of the image. (More details coming soon)
 
+**QR Code Monster**:
```

Comment:
Has this officially been added to the Installer?

Reply:
Will be adding in a separate PR